nengo_sphinx_theme/ext/autoautosummary.py

```diff
@@ -122,12 +148,76 @@ def patch_autosummary_import_by_name():
 
     def import_by_name(name, prefixes):
         # Filter out problematic prefixes
-        prefixes = [p for p in prefixes if p is None or not name.startswith(p)]
-        return orig_f(name, prefixes)
+        # TODO: are there any situations where we want prefixes?
```

Comment:
Not sure if there is anything we want to do about this TODO. This works fine in this repo, nengo core, and nengo-dl, so we weren't relying on the prefix functionality in any of those cases. But I'm not sure if there's a more permanent way to verify that we aren't relying on that functionality. Only thing that occurred to me is requiring that ``name`` contain ".", but that's not a reliable solution.

Reply:
If an import can't be found by name, we'll obviously get an error, so IMO that's just as good as making our own error e.g. if `name` doesn't contain ".". I would just change this to say e.g. "We currently do not allow prefixes to be used". Then in the future if we need this, we can add it back in.

Reply:
Yeah that sounds good